script/setup.ts

```diff
@@ -1,5 +1,8 @@
 import { setupConfigs } from "./setup/setupConfig";
 
+import { EventEmitter } from "events";
+EventEmitter.defaultMaxListeners = 20;
```

Comment:
adding this removed the warning you were getting while running scripts ?

Reply:
yes, I have not had any logs related to EventEmitter after